bin/dry-run.js

```diff
@@ -148,7 +148,7 @@ if (ignoredErrors.length) {
 }
 
 if (DUMP) {
-  const props = ['cid', 'minerId', 'participantAddress', 'inet_group', 'retrievalResult', 'fraudAssessment']
+  const props = ['cid', 'minerId', 'participantAddress', 'inet_group', 'retrievalResult', 'taskingEvaluation']
```

Comment:
should this not include `majorityEvaluation`, to represent the same range of information as the former `fraudAssessment`?
```suggestion
  const props = ['cid', 'minerId', 'participantAddress', 'inet_group', 'retrievalResult', 'taskingEvaluation', 'majorityEvaluation']
```

Reply:
I re-read the PR description and now see that this change in behavior might be intentional. Are you confident that all changes in behavior in this PR are intentional?

Reply:
Good catch, I'll accept your suggestion.

I'd like to point out that `dry-run` is a script we use internally only, and `DUMP` is an opt-in tool for debugging. I haven't used it for months. IMO, a breaking change here would have a very low impact and should be easy to fix.